### ISSUE TITLE
Trackman should fully replace engineering log 

### ISSUE THREAD

**Issue by echarlie** [wuvt/trackman#32]
_From @matthazinski on March 1, 2015 21:27_

It would be extremely useful if trackman could allow people to indicate:
- When PSAs were read (and which ones)
- When underwriting was read
- When weather was read
- When station promos were read (and which ones)
- Transmitter readings
- Operator sign on/off times

This could replace the physical log book

_Copied from original issue: wuvt/wuvt-site#21_

**Comment by echarlie** [comment]
_From @telnoratti on March 4, 2015 8:0_

It would be nice if trackman added engineering logs to the queue at the designated times.

**Comment by echarlie** [comment]
This is a generic-ish container for #34 and #35

**Comment by echarlie** [comment]
okay. trackman doesn't do anything for #35 right now, and we've accepted that. #34 is nice-to-have.

The actual engineering log is a record of who is responsible for the operation of station at the time of record. From brief conversations with kf4ylm, no digital log is sufficient for this, and when automation is on-air, the chief engineer or IT director is ostensibly responsible.

assuming trackman+airchecks are sufficient for determining a responsible party (independent of FCC operation log), this is completely unnecessary.

Closing.
